proto/api/registration/registration.proto

```diff
@@ -75,6 +75,12 @@ message JoinToken {
     int32 ttl = 2;
 }
 
+// CA Bundle of the server  
+message Bundle {
+    // ASN.1 DER data of the bundle.
+    bytes asn1Data = 1;
```

Comment:
nit: while this field name is descriptive, it feels a little cumbersome to have in the generated code... the datastore protobuf uses `ca_certs` field in its Bundle message... just a thought, not a blocker